unit-tests/syncer/test-ts-diff-fps.py

```diff
@@ -0,0 +1,258 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+import pyrealsense2 as rs
+from rspy import log, test
+
+
+# The timestamp jumps are closely correlated to the FPS passed to the video streams:
+# syncer expects frames to arrive every 1000/FPS milliseconds!
+fps_d = 100
+fps_c =  10
+gap_d = 1000 / fps_d
+gap_c = 1000 / fps_c
+domain = rs.timestamp_domain.hardware_clock
+
+def generate_depth_frame( frame_number, timestamp ):
+    """
+    """
+    global depth_profile, domain, pixels, depth_sensor
+    #
+    depth_frame = rs.software_video_frame()
+    depth_frame.pixels = pixels
+    depth_frame.stride = 640 * 2  # W * bpp
+    depth_frame.bpp = 2
+    depth_frame.frame_number = frame_number
+    depth_frame.timestamp = timestamp
+    depth_frame.domain = domain
+    depth_frame.profile = depth_profile
+    #
+    log.d( "-->", depth_frame )
+    depth_sensor.on_video_frame( depth_frame )
+
+def generate_color_frame( frame_number, timestamp ):
+    """
+    """
+    global color_profile, domain, pixels, color_sensor
+    #
+    color_frame = rs.software_video_frame()
+    color_frame.pixels = pixels
+    color_frame.stride = 640 * 2  # W * bpp
+    color_frame.bpp = 2
+    color_frame.frame_number = frame_number
+    color_frame.timestamp = timestamp
+    color_frame.domain = domain
+    color_frame.profile = color_profile
+    #
+    log.d( "-->", color_frame )
+    color_sensor.on_video_frame( color_frame )
```

Comment:
Can merge generate_depth_frame and generate_color_frame to generate_frame with parameter

Reply:
I could but I don't want to :)
There're lots of variables in each that're stream-dependent. I'd rather leave them simple.